install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -126,7 +126,7 @@ It may not be immediately obvious which SCC was matched by a pod, so the command
 above can be very useful in understanding the UID, supplemental groups, and
 SELinux relabeling in a live container.
 
-Any SCC with a strategy set to *RunAsAny* allows arbitrary values for that
+Any SCC with a strategy set to *RunAsAny* allows specific values for that
```

Comment:
yes it's intentional.

RunAsAny means the user can specify the uid, so saying it allows "specific" values makes more sense to me than "arbitrary".  Because elsewhere we use arbitray when saying "openshift runs your image w/ an arbitrary uid".  So i'm trying to straighten out the usage:

arbitrary - a randomly assigned value the user has no control over
specific - a value chosen by the user.

I think it makes sense if you read the full sentence:
"allows specific values for that strategy to be defined in the pod definition"  - the user can specify the value in the pod definition.

"allows arbitrary values for that strategy to be defined....." - this implies to me that i have no control over the value.